src/aave-v2/SupplyVaultBase.sol

```diff
@@ -104,17 +107,8 @@ abstract contract SupplyVaultBase is ISupplyVaultBase, ERC4626UpgradeableSafe, O
 
     /// EXTERNAL ///
 
-    /// @notice Sets the rewards recipient.
-    /// @dev Sets to address(0) to prevent MORPHO rewards from being transferred.
-    /// @param _recipient The new rewards recipient.
-    function setRewardsRecipient(address _recipient) external onlyOwner {
-        recipient = _recipient;
-        emit RewardsRecipientSet(_recipient);
-    }
-
     /// @notice Transfers the MORPHO rewards to the rewards recipient.
     function transferRewards() external {
```

Comment:
Why would the zero check need to be removed?

Reply:
It's already check before constructing the contract si I think it's just redundant